test/index.test.ts

```diff
@@ -1,8 +1,115 @@
 import { expect, it, describe } from "vitest";
-import {} from "../src";
+import {
+  bold,
+  blockquote,
+  boldAndItalic,
+  codeBlock,
+  heading,
+  hr,
+  image,
+  italic,
+  link,
+  list,
+  strikethrough,
+  table,
```

Comment:
After refactoring you can import `md` btw and use `md[key]` to simplify.

Reply:
done.